model-optimizer/mo/utils/cli_parser.py

```diff
@@ -89,12 +89,14 @@ def __call__(self, parser, namespace, values, option_string=None):
 
 class DeprecatedCanonicalizePathCheckExistenceAction(CanonicalizePathCheckExistenceAction):
     def __call__(self, parser, namespace, values, option_string=None):
-        super().__call__(parser, namespace, values, option_string)
         dep_msg = "Use of deprecated cli option {} detected. Option use in the following releases will be fatal. ".format(
             option_string)
         if 'tensorflow_use_custom_operations_config' in option_string:
             dep_msg += 'Please use --transformations_config cli option instead'
+        if 'mean_file' in option_string or 'mean_offset' in option_string:
+            dep_msg += 'Please use --mean_values cli option instead.'
         log.error(dep_msg, extra={'is_warning': True})
+        super().__call__(parser, namespace, values, option_string)
```

Comment:
why is this changed? Please move to the preliminary code line

Reply:
Because if we have error inside option processing, warning about deprecation will be missed, I think it is wrong, user should see deprecation warning in any way.